src/main/java/com/vint/shop/domain/Users.java

```diff
@@ -0,0 +1,70 @@
+package com.vint.shop.domain;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table (name = "users")
+public class Users {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "password_hash")
+    private String password_hash;
+
+    @Column(name = "first_name")
+    private String first_name;
+
+    @Column(name = "last_name")
+    private String last_name;
+
+    @Column(name = "email")
+    private String email;
+
+    @Column(name = "mobile")
+    private String mobile;
+
+    @Column(name = "role_id")
+    @OneToMany (fetch = FetchType.LAZY, mappedBy = "id")
+    private List<Roles> role_id;
```

Comment:
According to your database schema, it seems, the relation should be inverted: ONE Role for MANY Users

Reply:
made